cognosys-public/secured-visual-studio-pro-2015-win-ser-2012/autogen.yaml

```diff
@@ -0,0 +1,36 @@
+singleVm:
+  images:
+  - project: cognosys-public
+    name: cognosys-secured-visual-studio-pro-2015-win-ser-2012-12sep
+  firewallRules:
+  - protocol: TCP
+    port: '80'
+  - protocol: TCP
+    port: '443'
+  - protocol: TCP
+    port: '3389'
+  machineType:
+    defaultMachineType:
+      gceMachineType: n1-standard-4
+    minimum:
+      cpu: 2
+      ramGb: 7.5
+  bootDisk:
+    diskSize:
+      defaultSizeGb: 127
+      minSizeGb: 127
+    diskType:
+      defaultType: pd-standard
+  postDeploy:
+    connectButtonLabel: RDP
+    actionItems:
+    - heading: Next Steps To Connect
+      description: >-
+        Connecting to Instance:
+        <a href="http://secureanycloud.com/google-cloud-connect-using-rdp-to-windows-instance-on-gcp/">
+        Detailed Instructions for Connecting to the Instance
+        </a>
+  ipForwarding:
```

Comment:
Note that if we do not care about IP forwarding, we should drop this section instead of setting the default to Off. If this section is not defined, user won't be able to turn on IP forwarding.